frontend/webapp/reuseable-components/nodes-data-flow/builder.ts

```diff
@@ -133,13 +133,14 @@ export const buildNodesAndEdges = ({
           const actionSpec: ActionItem = typeof action.spec === 'string' ? JSON.parse(action.spec) : (action.spec as ActionItem);
 
           return createNode(`action-${index}`, 'base', centerColumnX, NODE_HEIGHT * (index + 1), {
+            id: action.id,
             type: OVERVIEW_ENTITY_TYPES.ACTION,
-            title: actionSpec.actionName,
+            status: 'healthy',
```

Comment:
I would move status type to type file as we going to reuse this in multiple places